common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -976,6 +976,14 @@
   "feature.wynntils.tradeMarketBulkSell.sellAll": "Sell All",
   "feature.wynntils.tradeMarketPriceConversion.description": "Extends entering prices from just a number to using math and having shorthand for Emeralds, Emerald Blocks, Liquid Emeralds, Liquid Emerald Stacks. Also adds the ability to include tax in the price.",
   "feature.wynntils.tradeMarketPriceConversion.name": "Market Price Conversion",
+  "feature.wynntils.tradeMarketPriceMatch.description": "Adds buttons to match lowest sell offer and highest buy offer in the trade market UI",
+  "feature.wynntils.tradeMarketPriceMatch.highestBuyOffer": "Highest Buy Offer",
+  "feature.wynntils.tradeMarketPriceMatch.highestBuyOfferTooltip": "Quick sells the item for the highest buy offer.",
+  "feature.wynntils.tradeMarketPriceMatch.lowestSellOffer": "Lowest Sell Offer",
+  "feature.wynntils.tradeMarketPriceMatch.lowestSellOfferTooltip": "Matches the price of the current lowest sell offer.",
```

Comment:
The two tooltips have different wordings, but they do basically the same thing but picks highest respectively the lowest price, yes? Can you adjust them to match, if so. Otherwise, explain better how they differ.